docs/developer/contributing.rst

```diff
@@ -230,9 +231,9 @@ conda. With cmder open in the pymedphys directory, run:
 
 .. code:: bash
 
-    conda install pymedphys --only-deps
-    pip install -e .
+    yarn bootstrap
```

Comment:
See the `bootstrap` command within `package.json`, this now handles dependencies, nbstripout, and a development install, as well as adding a hook so that before every commit it is confirmed that nbstripout is installed.